### PR TITLE
change global var from a function

### PR DESCRIPTION
**Pull request by Lex-2008**
Test that global variables are writeable from within a function.

### CODE REVIEW

**Comment by cwfitzgerald** [comment]
These two things really should be split into global variable reading and writing. Other than that, it looks good!

File: tests/correct/globalVariables.rock [hunk 1]
```diff
@@ -14,3 +14,11 @@ Give back X plus GlobalTwo
 
 put 6 into GlobalTwo
 say MyFuncTwo taking 2
+
+SetGlobal takes X
+put X into Global
+Give back X
```
Comment (marcinruszkiewicz) [inline tests/correct/globalVariables.rock]: This should read `Give back Global` if you want it to return 11. `X` is 1 at this point

Comment (Lex-2008) [reply tests/correct/globalVariables.rock]: Agreed

**Comment by Lex-2008** [comment]
Hard to say about splitting into separate tests - I would rather not do it, but if you say so... Done!

**Comment by dylanbeattie** [comment]
I've copied the additional test cases into the `globalVariables.rock` test on the main branch (e01a283fd4dbc1013e50103640e998ce3d9e370b), so I'm closing the PR.

**Pull request closed without merging**